### ISSUE TITLE
[Layout] Can't see or select panels under other layouts

### ISSUE THREAD

**Issue by VWoeltjen** [nasa/openmctweb#123]
When one panel obscures another completely in a Layout in Edit mode, you can't see (by hovering or other obvious gesture) that the obscured panel is present.

Would be nice to do something similar to what we do in Fixed Position here, with some combination of outlines and/or opacity which appear (either on hover or always) while editing, to allow the placement of these panels to be observed.

**Comment by charlesh88** [comment]
Definitely agree. I would definitely wait on this issue for the new edit mode: as discussed, we should move fixed position layout from Panels into Layouts, and we can fix this then.

**Comment by larkin** [comment]
I recall in a previous test-athon that this was not a problem-- when mousing over a panel that obscured another panel, the other panel would pop out from behind the panel and become selectable.  I'd describe the previous functionality as "when mousing over stacked objects, bring to focus the object that has the tightest bounds on the current mouse position."  With the latest testathon, it's no longer possible to do this.

I wasn't sure whether to reopen WTD-1383 (which I've now realized was a separate issue that occurred when drag or drop was active), or to file a new ticket.

**Comment by charlesh88** [comment]
Sprint planning 9/28/15: defer to new edit mode.

**Comment by charlesh88** [comment]
This should be fixed when we work on #1101.

**Comment by unlikelyzero** [comment]
I'm restating this as an enhancement that should be pursued
